frontend/src/components/worksheets/items/WorksheetItem.js

```diff
@@ -9,14 +9,19 @@ class WorksheetItem extends React.Component {
     constructor(props) {
         super(props);
         this.state = Immutable({});
+        this.rowRefs = {}; // Map of {elementId: ref}
     }
 
     capture_keys() {
         // Open worksheet in same tab
         Mousetrap.bind(
             ['enter'],
-            function(e) {
-                this.props.openWorksheet(this.refs['row' + this.props.subFocusIndex].props.uuid);
+            function() {
+                this.props.openWorksheet(
+                    this.rowRefs[
```

Comment:
We repeat this line four times -- can we refactor it into a function, just for readability?

Reply:
Which line do you refer to?

Reply:
```
const uuid = this.rowRefs[
                    `codalab-worksheet-item-${this.props.focusIndex}-subitem-${this.props.subFocusIndex}`
                ].current.props.uuid;
```

Reply:
Thanks for the advice. I have refactored the repeated lines